modules/rosa-sts-required-roles-and-policies.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// rosa_architecture/rosa-sts-about-iam-resources.adoc
+//
+:_content-type: CONCEPT
+[id="rosa-sts-required-roles-and-policies_{context}"]
+= Configuring ROSA cluster roles and policies
+
+Before creating a ROSA cluster, you must create the required Operator roles and set the correct OpenID Connect (OIDC) provider. You can create these roles by using the `rosa` command-line interface (CLI) tool or the ROSA guided installation in {cluster-manager-url}. You can create these roles manually or by using an automatic process that uses pre-defined names for these roles and polices.
+
+[id="rosa-sts-required-roles-and-policies-automatic_{context}"]
+== Automatic `ocm-role` resource creation
+
+You can use the automatic creation method in the CLI and from {cluster-manager} if you created an `ocm-role` resource with administrative permissions. Selecting this method creates the roles and policies that uses the default names.
+
+.Procedure
+
+* In the guided ROSA cluster installation in {cluster-manager}, select 'Auto mode' on step four, "Cluster roles and policies," to automatically create the roles and associate the policies by using the admin `ocm-role` resource.
++
+[NOTE]
+====
+If you use the ROSA guided installation on {cluster-manager}, you must have created an `ocm-role` resource with administrative permissions in the first step of the guided cluster installation. Without this role, you cannot use the automatic Operator role and policy creation option.
+====
+
+* In your preferred CLI, create the roles with the following command:
++
+[source,terminal]
+----
+rosa create account-role --mode auto
+----
++
+After creating these resources in your CLI or with the automatic processes, you have the following roles and policies:
```

Comment:
As mentioned previously, I think this concept information in the table would be useful for the reader to see prior to the procedure. It seems like a lot of concept information to have in a procedural step. In the current live publication, the resource descriptions and their associated policy files are covered and then an example creation procedure is provided after. What is the reason for changing this?

Additionally, the embedded JSON file references are being removed in this PR. These were added on request from the product team when I updated the STS installation section. Is there an explicit requirement to remove those now? If we do, do we need to send out a change management notification about that?